### ISSUE TITLE
Add News "total cards viewed" P3A question

### ISSUE THREAD

**Issue by DJAndries** [brave/brave-browser#22222]
## Description 

Add new News P3A question, for total cards viewed:

How many Brave News cards did you view in the past week?
`Brave.Today.WeeklyTotalCardViews`

0. None
1. 1
2. 2 to 10
3. 11 to 20
4. 21 to 40
5. 41 to 80
6. 81 to 100
7. 101 or more

## Miscellaneous Information:

Will raise PR soon.

**Comment by DJAndries** [comment]
cc @mattmcalister

**Comment by GeetaSarvadnya** [comment]
Verification is in-progress

```

Brave | 1.38.103 Chromium: 100.0.4896.127 (Official Build) beta (64-bit)
-- | --
Revision | ff0d0695743e65305d7194f9bd309e5e1c824aa0-refs/branch-heads/4896_88@{#4}
OS | Windows 10 Version 21H2 (Build 19044.1645)

```

**Comment by GeetaSarvadnya** [comment]
@DJAndries There is no metrics called `Brave.Today.WeeklyTotalCardViews` looks like it's mapping with the metrics `Brave.Today.WeeklyMaxCardViewsCount` which is showing up in brave://local-state. Could you update the issue along with the P3A wiki? Thanks!

**Comment by GeetaSarvadnya** [comment]
@DJAndries Logged issues above. Could you check them and update the response values if required so that I can proceed further with issues verification? Thanks!

**Comment by LaurenWags** [comment]
Adding `QA/Blocked` until the above is resolved

**Comment by DJAndries** [comment]
> @DJAndries There is no metrics called `Brave.Today.WeeklyTotalCardViews` looks like it's mapping with the metrics `Brave.Today.WeeklyMaxCardViewsCount` which is showing up in brave://local-state. Could you update the issue along with the P3A wiki if it's correct? Thanks!

WeeklyTotalCardViews is the correct metric name, however, I made a mistake by not using that metric name in the code. brave/brave-core#13360 will resolve this once merged. I will also request an uplift to 1.39.x.

**Comment by kjozwiak** [comment]
@GeetaSarvadnya https://github.com/brave/brave-core/pull/13373 has been merged into `1.39.x` so it should unblock https://github.com/brave/brave-browser/issues/22222#issuecomment-1129210319 once you get `1.39.109` or higher 👍

**Comment by GeetaSarvadnya** [comment]
Verification is in progress on `Sam sung Galaxy A52s version 11` running `Bravearmx64.apk_1.39.109`
